helper/resource/importstate/import_block_as_first_step_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package importstate_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+
+	"github.com/hashicorp/terraform-plugin-testing/internal/testing/testprovider"
+	"github.com/hashicorp/terraform-plugin-testing/internal/testing/testsdk/providerserver"
+	"github.com/hashicorp/terraform-plugin-testing/knownvalue"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/hashicorp/terraform-plugin-testing/tfjsonpath"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+
+	r "github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func Test_ImportBlock_AsFirstStep(t *testing.T) {
+	t.Parallel()
+
+	r.UnitTest(t, r.TestCase{
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.SkipBelow(tfversion.Version1_5_0), // ImportBlockWithId requires Terraform 1.5.0 or later
+		},
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"examplecloud": providerserver.NewProviderServer(testprovider.Provider{
+				Resources: map[string]testprovider.Resource{
+					"examplecloud_container": examplecloudResource(),
+				},
+			}),
+		},
+		Steps: []r.TestStep{
+			{
+				ResourceName:    "examplecloud_container.test",
+				ImportStateId:   "examplecloud_container.test",
+				ImportState:     true,
+				ImportStateKind: r.ImportBlockWithId,
+				// ImportStateVerify: true,
```

Comment:
I think this is already on your radar, but we should consider adding some explicit validation to TestStep to make it clear which fields you're allowed to use / will work together.

is this commented out to show that it's not possible to use here?

Reply:
> we should consider adding some explicit validation to TestStep to make it clear which fields you're allowed to use / will work together

💯. An upcoming pull request.

> is this commented out to show that it's not possible to use here?

This comment disappears in the PRs stacked atop this one, where we introduce `ImportPlanVerify`.